js/viewer/annotation/annotation-svg.js

```diff
@@ -246,24 +255,137 @@ function AnnotationSVG(parent, id, imgWidth, imgHeight, scale, ignoreReferencePo
+     * @param {object} node
+     * @return {styleAttributeString}
+     */
+    this.getStyleAttributes = function (node) {
+
+        console.log('styleProperties', node);
+        styleAttributeList = ['alignment-baseline', 'baseline-shift', 'clip', 'clip-path', 'clip-rule', 'color', 'color-interpolation', 'color-interpolation-filters', 'color-profile', 'color-rendering', 'cursor', 'direction', 'display', 'dominant-baseline', 'enable-background', 'fill', 'fill-opacity', 'fill-rule', 'filter', 'flood-color', 'flood-opacity', 'font-family', 'font-size', 'font-size-adjust', 'font-stretch', 'font-style', 'font-variant', 'font-weight', 'glyph-orientation-horizontal', 'glyph-orientation-vertical', 'image-rendering', 'kerning', 'letter-spacing', 'lighting-color', 'marker-end', 'marker-mid', 'marker-start', 'mask', 'opacity', 'overflow', 'pointer-events', 'shape-rendering', 'stop-color', 'stop-opacity', 'stroke', 'stroke-dasharray', 'stroke-dashoffset', 'stroke-linecap', 'stroke-linejoin', 'stroke-miterlimit', 'stroke-opacity', 'stroke-width', 'text-anchor', 'text-decoration', 'text-rendering', 'transform', 'transform-origin', 'unicode-bidi', 'vector-effect', 'visibility', 'word-spacing', 'writing-mode'];
+
+        styleAttributeString = '';
+
+        for (let i = 0; i < styleAttributeList.length; i++) {
+            if (node.getAttribute(styleAttributeList[i])) {
+                value = node.getAttribute(styleAttributeList[i]);
+                styleAttributeString += styleAttributeList[i] + ':' + value + ';';
+            }
+        }
+
+        return styleAttributeString;
+    }
+
+    /**
+     * Will return an object of the style string
+     * @param {string} styleString
+     * @return {styleObject}
+     */
+    this.styleStringToObject = function (styleString) {
+        styleString = styleString.replace(/\s/g, '');
+        if (styleString[styleString.length - 1] == ';') {
+            styleString = styleString.slice(0, styleString.length - 1);
+        }
+
+        var styleObject = {};
+
+        styleString = styleString.split(';');
+
+        for (i = 0; i < styleString.length; i++) {
+            var property = styleString[i].split(':')[0];
+            var value = styleString[i].split(':')[1];
+            styleObject[property] = value;
+        }
+
+        return styleObject;
+    }
+
+    /**
+     * Will return a string of the style object, opposite of styleStringToObject function
+     * @param {object} styleObject
+     * @return {styleString}
+     */
+    this.styleObjectToString = function (styleObject) {
+        var styleString = '';
+
+        for (let key in styleObject) {
+            styleString += key.toString() + ':' + styleObject[key].toString() + ';';
+        }
+
+        // remove the last ';'
+        styleString = styleString.slice(0, styleString.length - 1);
+
+        return styleString;
+    }
+
+    /**
+     * Take the properties from the parent style and appends them to the child node
```

Comment:
If they are missing from the child, right?

Reply:
Yes, I have updated the comment too